addons/xterm-addon-search/src/SearchAddon.ts

```diff
@@ -220,18 +239,28 @@ export class SearchAddon implements ITerminalAddon {
    * started on an earlier line then it is skipped since it will be properly searched when the terminal line that the
    * text starts on is searched.
    * @param term The search term.
-   * @param row The line to  start the search from.
-   * @param col The column to start the search from.
+   * @param {ISearchPosition} position The position to start the search.
```

Comment:
Can skip the type annotations since we're in TS

```suggestion
   * @param position The position to start the search.
```